.github/workflows/reusable_pull_request.yml

```diff
@@ -1,8 +1,23 @@
 name: Pull Request
 
 on:
-  pull_request:
-    types: [opened, reopened, synchronize, ready_for_review]
+  workflow_call:
+    inputs:
+      enable_unit_tests:
+        type: boolean
+        description: "Boolean to enable the unit tests job. Defaults to true."
+        required: true
+        default: true
```

Comment:
Seems like an odd mix, if there's a default is the input actually required?

Reply:
Yeah the idea is to have each job be configurable and by default all jobs are enabled. So for most repos they just call this workflow and have to pass no inputs. Now some repos might want to turn on/off certain jobs and they would pass inputs.

Reply:
Right, so inputs aren't `required` then?

Reply:
I see what you mean. Yeah removed the `required`